.github/workflows/docker-build-and-push.yaml

```
@@ -0,0 +1,34 @@
+  push:
+    branches: [ main ]
+   
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        
+      - name: Authenticate to Google Cloud
+        uses: google-github-actions/auth@v1
```

actionlint (1.7.4)

the runner of "google-github-actions/auth@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)